perl/p5-tcl-ptk/Portfile

```diff
@@ -4,7 +4,7 @@ PortSystem          1.0
 PortGroup           perl5 1.0
 
 perl5.branches      5.26 5.28 5.30
-perl5.setup         Tcl-pTk 1.02
+perl5.setup         Tcl-pTk 1.06 ../../authors/id/C/CA/CAC
```

Comment:
I'm curious why adding the CPAN dir was added: was it necessary to add it/better to specify it than not to?

Reply:
By adding this relative path, the port downloads the distribution from the author's upload directory so the distfile is accessible almost immediately after uploading to PAUSE.  Without this you need to wait until the newly uploaded distribution is indexed by MetaCPAN which, as has been  discussed, can be several hours.

In this instance the download URL is:
`https://cpan.metacpan.org/authors/id/C/CA/CAC/Tcl-pTk-1.06.tar.gz`
which is what MetaCPAN uses in its download link on the module home page.

This also is helpful when downloads fail because of indexing failure due to a malformed distribution or one where the version has been incorrectly incremented.

I generally leave this off in a new port but add it in if I run into a download problem.  Either way should work eventually.